websites/T/Trakt/presence.ts

```diff
@@ -0,0 +1,142 @@
+const presence = new Presence({
+		clientId: "968880591003082783"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/7V0kxEb.png",
+			startTimestamp: browsingTimestamp
+		},
+		path = document.location.pathname.toLowerCase();
+
+	if (path == "/") presenceData.details = "Browsing main page...";
+	else if (path == "/dashboard") presenceData.details = "Browsing dashboard...";
+	else if (path.includes("/users")) {
+		const username = document.querySelector(
+			"#avatar-wrapper > h1 > a"
+		).textContent;
+		if (path.includes("/history"))
+			presenceData.details = `Browsing ${username}'s history`;
+		else if (path.includes("/progress"))
+			presenceData.details = `Browsing ${username}'s progress`;
+		else if (path.includes("/collection"))
+			presenceData.details = `Browsing ${username}'s collection`;
+		else if (path.includes("/ratings"))
+			presenceData.details = `Browsing ${username}'s ratings`;
+		else if (path.includes("/lists"))
+			presenceData.details = `Browsing ${username}'s lists`;
+		else if (path.includes("/comments"))
+			presenceData.details = `Browsing ${username}'s comments`;
+		else if (path.includes("/network" || "/network/friends"))
+			presenceData.details = `Browsing ${username}'s friends`;
+		else presenceData.details = `Browsing ${username}'s profile`;
+	} else if (path.includes("/shows")) {
+		if (path.includes("/episodes/")) {
+			presenceData.details = "Browsing episode...";
+			presenceData.state = `${
+				document
+					.querySelector(
+						"#summary-wrapper > div.container.summary > div > div > div.col-md-10.col-md-offset-2.col-sm-9.col-sm-offset-3.mobile-title > h2 > a:nth-child(1)"
+					)
+					.textContent.split(":")[0]
+			} - 
+		${
+			document.querySelector(
+				"#summary-wrapper > div.container.summary > div > div > div.col-md-10.col-md-offset-2.col-sm-9.col-sm-offset-3.mobile-title > h1 > span.main-title-sxe"
+			).textContent
+		}`;
+		} else if (path.includes("/seasons/")) {
+			presenceData.details = "Browsing season...";
```

Comment:
Remove `...` on all references too please
```suggestion
			presenceData.details = "Browsing season";
```